`pkgx pod` behaving erratically due to ncurses

Repro:

1. Create an empty folder
2. Create an empty file called SwiftUIIntrospect.podspec and copy the following in:
	```ruby
	Pod::Spec.new do |spec|
	  spec.name = 'SwiftUIIntrospect'
	  spec.version = '1.4.0-beta.4'
	  spec.license = { type: 'MIT' }
	  spec.homepage = 'https://github.com/siteline/swiftui-introspect'
	  spec.author = 'David Roman'
	  spec.summary = 'Introspect underlying UIKit/AppKit components from SwiftUI.'
	  spec.source = {
	    git: 'https://github.com/siteline/swiftui-introspect.git',
	    tag: spec.version
	  }
	
	  spec.source_files = 'Sources/**/*.swift'
	
	  spec.swift_version = '6.0'
	  spec.ios.deployment_target = '13.0'
	  spec.tvos.deployment_target = '13.0'
	  spec.osx.deployment_target = '10.15'
	  spec.visionos.deployment_target = '1.0'
	end
	```
3. In terminal, go to the folder and run:
	```bash
	pkgx pod spec lint SwiftUIIntrospect.podspec
	```

Output:

```
 -> SwiftUIIntrospect (1.4.0-beta.4)
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Building targets in dependency order
    - NOTE  | xcodebuild:  note: Target dependency graph (3 targets)
    - NOTE  | [iOS] xcodebuild:  2025-09-10 14:39:03.345 appintentsmetadataprocessor[65366:24553129] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | xcodebuild:  note: Signing static framework with --generate-pre-encrypt-hashes (in target 'Pods-App' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  2025-09-10 14:39:05.041 appintentsmetadataprocessor[65417:24553573] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [tvOS] xcodebuild:  2025-09-10 14:39:11.620 appintentsmetadataprocessor[65538:24554364] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [tvOS] xcodebuild:  2025-09-10 14:39:13.265 appintentsmetadataprocessor[65587:24554744] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - ERROR | [OSX] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  note: Using codesigning identity override:
    - NOTE  | xcodebuild:  <module-includes>:345:9: note: in file included from <module-includes>:345:
    - NOTE  | xcodebuild:  |         `- note: in file included from <module-includes>:345:
    - NOTE  | xcodebuild:  /Applications/Xcode-26.0.0-Release.Candidate.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/curses.h:2918:10: note: in file included from /Applications/Xcode-26.0.0-Release.Candidate.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/curses.h:2918:
    - NOTE  | xcodebuild:  |          `- note: in file included from /Applications/Xcode-26.0.0-Release.Candidate.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/curses.h:2918:
    - ERROR | [OSX] xcodebuild:  /Users/davdroman/.pkgx/invisible-island.net/ncurses/v6.4.0/include/unctrl.h:61:54: error: conflicting types for 'unctrl'
    - NOTE  | [OSX] xcodebuild:  |                                      `- note: previous declaration is here
    - NOTE  | [OSX] xcodebuild:  |                                                      `- error: conflicting types for 'unctrl'
    - NOTE  | [OSX] xcodebuild:  <unknown>:0: error: failed to emit precompiled module '/Users/davdroman/Library/Developer/Xcode/DerivedData/App-bkyxxehmcquazcfeoxfioxdcdpql/Build/Intermediates.noindex/SwiftExplicitPrecompiledModules/Darwin-5AX429P2UCTDOJAUCCX0BI9T4.pcm' for module map '/Applications/Xcode-26.0.0-Release.Candidate.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/Darwin.modulemap'
    - NOTE  | [visionOS] xcodebuild:  2025-09-10 14:39:22.529 appintentsmetadataprocessor[65741:24555844] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [visionOS] xcodebuild:  2025-09-10 14:39:24.384 appintentsmetadataprocessor[65827:24556337] warning: Metadata extraction skipped. No AppIntents.framework dependency found.

Analyzed 1 podspec.

[!] The spec did not pass validation, due to 2 errors.
```

Seems related to ncurses:

```
    - ERROR | [OSX] xcodebuild:  /Users/davdroman/.pkgx/invisible-island.net/ncurses/v6.4.0/include/unctrl.h:61:54: error: conflicting types for 'unctrl'
    - NOTE  | [OSX] xcodebuild:  |                                      `- note: previous declaration is here
    - NOTE  | [OSX] xcodebuild:  |                                                      `- error: conflicting types for 'unctrl'
```

Note: running `rm -rf /Users/davdroman/.pkgx/invisible-island.net` and retrying makes it succeed:

```
 -> SwiftUIIntrospect (1.4.0-beta.4)
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Building targets in dependency order
    - NOTE  | xcodebuild:  note: Target dependency graph (3 targets)
    - NOTE  | xcodebuild:  ld: warning: search path '/Users/davdroman/.pkgx/invisible-island.net/ncurses/v6.4.0/lib' not found
    - NOTE  | [iOS] xcodebuild:  2025-09-10 14:44:51.716 appintentsmetadataprocessor[66076:24563486] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | xcodebuild:  note: Signing static framework with --generate-pre-encrypt-hashes (in target 'Pods-App' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  2025-09-10 14:44:53.322 appintentsmetadataprocessor[66132:24563918] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [tvOS] xcodebuild:  2025-09-10 14:44:59.500 appintentsmetadataprocessor[66255:24564695] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [tvOS] xcodebuild:  2025-09-10 14:45:01.059 appintentsmetadataprocessor[66301:24565072] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | xcodebuild:  note: Using codesigning identity override:
    - NOTE  | [OSX] xcodebuild:  2025-09-10 14:45:07.674 appintentsmetadataprocessor[66450:24566023] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [OSX] xcodebuild:  2025-09-10 14:45:09.452 appintentsmetadataprocessor[66495:24566380] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [visionOS] xcodebuild:  2025-09-10 14:45:16.105 appintentsmetadataprocessor[66633:24567258] warning: Metadata extraction skipped. No AppIntents.framework dependency found.
    - NOTE  | [visionOS] xcodebuild:  2025-09-10 14:45:17.983 appintentsmetadataprocessor[66722:24567712] warning: Metadata extraction skipped. No AppIntents.framework dependency found.

Analyzed 1 podspec.

SwiftUIIntrospect.podspec passed validation.
```

--- a/projects/cocoapods.org/package.yml
+++ b/projects/cocoapods.org/package.yml
@@ -25,9 +25,9 @@ build:
     - gem build cocoapods.gemspec
     - gem install cocoapods-{{version}}.gem
 
-    - run: |
-        mv {{prefix}}/bin/* .
-        cp $SRCROOT/props/proxy {{prefix}}/bin/pod
+    - run:
+        - mv {{prefix}}/bin/* .
+        - cp $SRCROOT/props/proxy {{prefix}}/bin/pod
       working-directory: ${{prefix}}/gems/bin
 
 provides:

--- a/projects/cocoapods.org/proxy
+++ b/projects/cocoapods.org/proxy
@@ -2,4 +2,10 @@
 export GEM_HOME="$(cd "$(dirname "$0")"/.. && pwd)"
 export GEM_PATH="$GEM_HOME"
 export PATH="$GEM_HOME/gems/bin:$PATH"
+
+# Conflicting with Apple's ncurses is bad. See:
+# https://github.com/pkgxdev/pantry/issues/10454
+
+export CPATH="$(echo $CPATH | tr ':' '\n' | grep -v 'invisible-island.net/ncurses' | tr '\n' ':')"
+
 exec "$GEM_HOME"/gems/bin/pod "$@"

